### PR TITLE
New Boss: Phosani's Nightmare

### PR DESCRIPTION
**Pull request by Not-Jayden**

### CODE REVIEW

File: __tests__/hiscores.test.ts [hunk 1]
```diff
@@ -212,6 +213,7 @@ test('Parse CSV to json', () => {
       krilTsutsaroth: { rank: 120, score: 2981 },
       mimic: { rank: 1, score: 109 },
       nightmare: { rank: 3, score: 22666 },
+      phosanisNightmare: { rank: 2, score: 84},
```
Comment (Not-Jayden) [inline __tests__/hiscores.test.ts:216]:
```suggestion
      phosanisNightmare: { rank: 2, score: 84 },
```

**Comment by molo-pl** [comment]
/nit I think you should also update README.md, e.g. as per 6b31e8b08c3bcdac9a00004caca31ecf51f0e2b2

**Comment by Not-Jayden** [comment]
> /nit I think you should also update README.md, e.g. as per [6b31e8b](https://github.com/maxswa/osrs-json-hiscores/commit/6b31e8b08c3bcdac9a00004caca31ecf51f0e2b2)

I did but apparently I didn't save it 🤦‍♂️

**Comment by Not-Jayden** [comment]
Updated now

**Review by maxswa — APPROVED** [review]
Thanks for another contribution!

**Pull request merged**